### PR TITLE
RFC: Add Julia 1.0 announcement post

### PR DESCRIPTION
**Pull request by mbauman**
Do not merge until we have binaries to download. Please do not disseminate this PR itself to press/twitter/social media/etc.

### CODE REVIEW

File: blog/_posts/2018-08-08-one-point-zero.md [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+* **Composable**: Julia’s packages naturally work well together. Matrices of unit
+  quantities, or data table columns of currencies and colors, just work — and with good
+  performance.
+
+Try Julia by [downloading version 1.0 now](https://julialang.org/downloads/). If you’re
+upgrading code from Julia 0.6 or earlier, we encourage you to first use the transitional 0.7
+release, which includes deprecation warnings to help guide you through the upgrade process.
+Once your code is warning-free, you can change to 1.0 without any functional changes. The
+package ecosystem is in the midst of taking advantage of this stepping stone and many
+packages are already 1.0-compatible! This includes ...
```
Comment (mbauman) [inline blog/_posts/2018-08-08-one-point-zero.md]: TODO: add list of packages that are already 1.0 compatible.

File: blog/_posts/2018-08-08-one-point-zero.md [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+  `row.version` accesses the `version` field of a single row.
+
+* Julia’s own optimizer has gotten smarter in more ways than we can list here, but a few
+  highlights are worth mentioning. The optimizer can now propagate constants through
+  function calls, allowing much better dead-code elimination and static evaluation than
+  before. The compiler is also much better at avoiding allocation of short-lived wrappers
+  around long-lived objects, which frees programmers to use convenient high-level
+  abstractions without performance costs.
+
+* Parametric types constructors are now always called with the same syntax as they are
```
Comment (JeffBezanson) [inline blog/_posts/2018-08-08-one-point-zero.md]: types -> type

File: blog/_posts/2018-08-08-one-point-zero.md (hunk 1 above)
Comment (ViralBShah) [reply blog/_posts/2018-08-08-one-point-zero.md]: I assume this will be a URL, so that we can keep updating it without changing the post.

Comment (mbauman) [reply blog/_posts/2018-08-08-one-point-zero.md]: I had been thinking about an immediate list, but a dynamic page is even better.  The best answer is an output from NewPkgEval, but I don't want to put _anything_ more on Keno's plate.  I'll look into what we can do here.

Comment (Keno) [reply blog/_posts/2018-08-08-one-point-zero.md]: I can give you a JSON file with the NewPkgEval results. You can do whatever with that.

Comment (mbauman) [reply blog/_posts/2018-08-08-one-point-zero.md]: That'd be great.  I think we can create a simple page that simply lists the passing packages, sorted by their number of stars.

File: blog/_posts/2018-08-08-one-point-zero.md [hunk 3]
```diff
@@ -0,0 +1,161 @@
...
+
+* Named tuples are a new language feature which make representing and accessing data by name
+  efficient and convenient. You can, for example, represent a row of data as `row =
+  (name="Julia", version=v"1.0.0", releases=8)` and access the `version` column as
+  `row.version` with the same performance as the less convenient `row[2]`.
+
+* The dot operator can now be overloaded, allowing types to use the `obj.property` syntax
+  for meanings other than getting and setting struct fields. This is especially useful for
+  smoother interop with class-based languages such as Python and Java. Property accessor
+  overloading also allows the the syntax for getting a column of data to match named tuple
```
Comment (cormullion) [inline blog/_posts/2018-08-08-one-point-zero.md]: "the the"

**Comment by ViralBShah** [comment]
We probably want the reception video linked in this blog (or at least the 1.0 release video).

https://www.youtube.com/watch?v=1jN5wKvN-Uk

File: blog/_posts/2018-08-08-one-point-zero.md [hunk 4]
```diff
@@ -0,0 +1,161 @@
...
+
+The single most significant new feature in Julia 1.0, of course, is a commitment to language
+API stability: code you write for Julia 1.0 will continue to work in Julia 1.1, 1.2, etc.
+The language is “fully baked.” The core language devs and community alike can focus on
+packages, tools, and new features built upon this solid foundation.
+
+But Julia 1.0 in not just about stability, it also introduces several new, powerful and
+innovative language features. Some of the new features since version 0.6 include:
+
+* A brand new built-in [package manager](https://docs.julialang.org/en/latest/stdlib/Pkg/)
```
Comment (mbauman) [inline blog/_posts/2018-08-08-one-point-zero.md]: It'd be nice to use the `en/v1.0` permalink here.  We don't seem to have the 1.0 manual available yet, though. :-\

Comment (StefanKarpinski) [reply blog/_posts/2018-08-08-one-point-zero.md]: We can just fix that up after the fact.

**Comment by StefanKarpinski** [comment]
Are we ready to go with this or should we wait until the website is updated as well?

**Comment by mbauman** [comment]
I think it'd be good to wait for the 1.0 binaries to be posted. Otherwise the landing page of julialang.org will still say "Download 0.7".

**Comment by ararslan** [comment]
See #107

**Comment by mbauman** [comment]
Ok, I'm happy with this now!

File: blog/_posts/2018-08-08-one-point-zero.md [hunk 5]
```diff
@@ -0,0 +1,167 @@
...
+changes, see the [0.7 NEWS file](https://docs.julialang.org/en/release-0.7/NEWS/). In our
+original [“Why We Created Julia” blog
+post](https://julialang.org/blog/2012/02/why-we-created-julia) in 2012, we wrote
+
+> It’s not complete, but it’s time for a 1.0 release—the language we’ve created is called
+> [Julia](https://julialang.org).
+
+We may have jumped the gun a bit with mentioning an impending 1.0 release, but the time has
+finally arrived and it is a heck of a release. We are truly proud of what’s been
+accomplished by the thousands of people who have contributed in so many ways to this truly
```
Comment (rfourquet) [inline blog/_posts/2018-08-08-one-point-zero.md]: two "truly" a bit close to each other, or was it on purpose?

Comment (StefanKarpinski) [reply blog/_posts/2018-08-08-one-point-zero.md]: No, it was an accident... better word choices for either one appreciated.

**Pull request merged**